downstream/modules/platform/ref-hub-variables.adoc

```diff
@@ -200,6 +200,32 @@ This must be reachable from client machines.
 | *`sso_use_https`* | If Single Sign On uses https.
 
 Default = `true`
+| *`sso_console_admin_username`* | SSO admin username.
+
+Default = `admin`
+| *`sso_ssl_validate_certs`* | 
+
+Default = `true`
+| *`sso_custom_keystore_file`* | Customer-provided keystore for SSO.
+
+| *`sso_keystore_file_remote`* |  Set to `true` if the customer-provided keystore is on a remote node.
+
+Default = `false`
+| *`sso_keystore_name`* | Name of keystore for SSO.
+
+Default = `ansible-automation-platform`
+| *`sso_automation_platform_realm`* | 
+
+Default = `ansible-automation-platform`
+| *`sso_automation_platform_realm_displayname`* | Display name for the realm.
+
+Default = `Ansible Automation Platform`
+| *`sso_automation_platform_login_theme`* | Path to the theme files.
+
+Default = `ansible-automation-platform`
+| *`sso_http_port`* or *`sso_https_port`* | IP or routable hostname for SSO.
```

Comment:
I would make these separate entries.

Reply:
@ianf77 Satoe isn't sure whether these variables are necessary. I'm commenting them out in case they do become necessary.